shared/chat/conversations-list/index.desktop.js

```diff
@@ -11,30 +11,28 @@ import type {InboxState} from '../../constants/chat'
 
 const AddNewRow = ({onNewChat}: Props) => (
   <div
-    style={{...globalStyles.flexBoxRow, ...globalStyles.clickable, minHeight: 48, justifyContent: 'center', alignItems: 'center', flexShrink: 0}}
-    onClick={() => onNewChat()}>
-    <Icon type='iconfont-new' style={{color: globalColors.blue, marginRight: 9}} />
-    <Text type='BodyBigLink'>New chat</Text>
+    style={{...globalStyles.flexBoxRow, alignItems: 'center', flexShrink: 0, justifyContent: 'center', minHeight: 48}}>
+    <div style={{...globalStyles.flexBoxRow, ...globalStyles.clickable, alignItems: 'center', justifyContent: 'center'}} onClick={onNewChat}>
+      <Icon type='iconfont-new' style={{color: globalColors.blue, marginRight: 9}} />
+      <Text type='BodyBigLink'>New chat</Text>
+    </div>
   </div>
 )
 
-const rowBorderColor = (idx: number, lastParticipantIndex: number, hasUnread: boolean, isSelected: boolean) => {
+function rowBackgroundColor (hasUnread: boolean, isSelected: boolean) {
+  return isSelected ? globalColors.white : hasUnread ? globalColors.darkBlue : globalColors.darkBlue4
+}
+
+// All this complexity isn't great but the current implmentation of avatar forces us to juggle all these colors and
+// forces us to explicitly choose undefined/the background/ etc. This can be cleaned up when avatar is simplified
+function rowBorderColor (idx: number, lastParticipantIndex: number, hasUnread: boolean, isSelected: boolean) {
   // Not the most recent? Don't color
-  if (idx !== lastParticipantIndex) {
+  if (idx === lastParticipantIndex) {
```

Comment:
or even if it calculated these as `const` and then used them below. I just find L35 kinda hard to understand